### ISSUE TITLE
Solute options not read if moving fast

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molarity#156]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Firefox
**Problem description**
For https://github.com/phetsims/QA/issues/436
**Steps to reproduce**
For https://github.com/phetsims/QA/issues/436
Found with NVDA on firefox. When you hold the arrow key down to scroll through the solute options quickly, the screen reader stops reading and does not read the end option.  If you go one at a time this is not an issue.

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Molarity‬
URL: https://phet-dev.colorado.edu/html/molarity/1.5.0-dev.23/phet/molarity_en_phet.html
Version: 1.5.0-dev.23 2019-10-04 22:27:51 UTC
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:69.0) Gecko/20100101 Firefox/69.0
Language: en-US
Window: 1536x750
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0
GLSL: WebGL GLSL ES 1.0
Vendor: Mozilla (Mozilla)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
Thanks for pointing out @KatieWoe. I tried this out on NVDA and Firefox on my machine and saw the same behavior.

I also see the same behavior with this example from W3 outside of PhET https://www.w3.org/TR/wai-aria-practices/examples/listbox/listbox-collapsible.html

While good to know, I don't think this is worth any debugging time. Especially since subsequent arrow key presses work just fine. But @twant if you want you could run it by @terracoda to make sure this is OK.

**Comment by terracoda** [comment]
@twant, @KatieWoe and @jessegreenberg I feel this not a bug, but just the nature of moving fast with a screen reader.

The user doesn't get everything when they move fast.

**Comment by twant** [comment]
@terracoda @jessegreenberg Thanks for the advice! It sounds like this one is okay to close for now, but please let me know if I should reopen it for any reason, @KatieWoe!
